recipe/meta.yaml

```diff
@@ -79,7 +79,7 @@ outputs:
         - pandas >=1.2
     test:
       requires:
-        - pip
+        - pip <24.2
```

Comment:
That works. Another option is to or true the pip check call.
A comment as to why would help though.

Reply:
sure, I added the upper bound pinning only for `osx-arm64`, and added a comment